### ISSUE TITLE
Bulk actions for Problem list

### ISSUE THREAD

**Issue by Floppy** [manyfold3d/manyfold#2295]
It would be very useful to be able to perform actions in bulk on the problem list. So, we'll need some sort of multi-select, and buttons to perform the same action on everything selected.

**Comment by disconn3ct** [comment]
I think it should go beyond just the problem list. I would love to see a general bulk edit option.

For example, I just finished a bunch of esp32 mounts. Uploaded about 10, printed 3, liked 1. Now I need to go back add an 'esp32' and 'printed' tags to those projects, remove '!new' etc. I was hoping to find a list similar to the non-graphics 'main' page, with checkboxes and bottom-edit-columns. Maybe I'm doing this the hard way, but what I ended up with is: "open model, click edit, click tags, update them (lots of forced clicking on text), click save, hit back (edit page), hit back (results), repeat."

I would ideally just check them off a list, select their tags and hit save.
